lib/limiter/connlimiter.go

```diff
@@ -29,7 +29,7 @@ import (
 // ConnectionsLimiter is a network connection limiter and tracker
 type ConnectionsLimiter struct {
 	*connlimit.ConnLimiter
-	*sync.Mutex
+	sync.Mutex
 	connections    map[string]int64
 	maxConnections int64
```

Comment:
```suggestion
	maxConnections int64

	mu          sync.Mutex
	connections map[string]int64
```

or something along those lines, to make it clear that the mutex is only protecting the map?